deployments/yunikorn-application/sample-yunikorn-app.yaml

```diff
@@ -20,5 +20,19 @@ kind: Application
 metadata:
   name: example
 spec:
+  appId: "app0001"
```

Comment:
I agree. But I think we should publish the appID. Right now, the definition does not have any appID field. I think it will be very useful to publish it. Right now the used doesn't know the appId after submitting the CRD. He/she will have to check the webUI or to recall it from the docs/code that it is in the form of namespace-name.

Reply:
Or we should move `AppID` from spec field to status field ?

Reply:
Yes, I think we can do that.